src/Apps/Order/Routes/Payment/index.tsx

```diff
@@ -265,7 +283,8 @@ export class PaymentRoute extends Component<PaymentProps, PaymentState> {
   }
 
   private onMutationError(errors) {
-    console.error(errors)
+    console.error("Order/Routes/Payment/index.tsx", errors)
+    this.setState({ isComittingMutation: false, isErrorModalOpen: true })
```

Comment:
The original ticket had a requirement to also start displaying the formatted errors we're already getting back from metaphysics during the `createCreditCard` error (see [this file](https://github.com/artsy/emission/blob/c006850dc1383f4f05e8e25ef127af2c35f01066/src/lib/Components/Bidding/Screens/Registration.tsx#L164) for how we do it in emission).

If you think that is out-of-scope of this PR, we should create a sub-ticket to track that requirement. If you think it's small enough to add here that's 👍 too.

Reply:
I think I should be able to handle it as part of this story - I just missed the last sentence in the ticket.